database/seeders/DatabaseSeeder.php

```diff
@@ -4,14 +4,19 @@
 
 namespace Database\Seeders;
 
+use App\Models\City;
+use App\Models\CityAlternativeName;
 use Illuminate\Database\Seeder;
 
 class DatabaseSeeder extends Seeder
 {
-    /**
-     * Seed the application's database.
-     */
     public function run(): void
     {
+        City::factory()->count(20)->create();
+        CityAlternativeName::factory()->count(10)->create();
```

Comment:
It seems to me that this should be removed.

Reply:
I think you should remove it for now.

Reply:
Done.